.github/workflows/publish_to_hex_pm.yml

```diff
@@ -0,0 +1,46 @@
+name: "Publish to Hex.pm"
+
+on:
+  workflow_run:
+    workflows:
+      - Test template
+    branches:
+      - master
+    types:
+      - completed
+  workflow_dispatch:
+
+env:
+  OTP_VERSION: 23.1.4
+  ELIXIR_VERSION: 1.11.2
+
+jobs:   
+  publish:
+    runs-on: ubuntu-latest
```

Comment:
When using `workflow_run`, you also need to add a conditional to also be able to re-run the job manually:

```
if: ${{ github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success' }}
```

Reply:
Good to know that @olivierobert, I have fixed that in https://github.com/nimblehq/elixir-templates/pull/63/commits/92806e395a01744416ee762ae04aaea7341720e0